apps/site/components/withNavBar.tsx

```diff
@@ -23,6 +24,15 @@ const WithNavBar: FC = () => {
 
   return (
     <div>
+      <Button
```

Comment:
I think this should be a link instead of a button, something basic (like GitHub) could be considered for styling